blockmanager.go

```diff
@@ -1012,10 +1016,6 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 				log.Debugf("Got response for headers at "+
 					"height=%v, only at height=%v, stashing",
 					startHeight, curHeight)
-
-				queryResponses[checkPointIndex] = r
```

Comment:
So how are out of order responses handled now? Do we even needs this block any longer?

Reply:
It is handled below by always writing the response to the map, then iterating over the map. If statement kept to keep the logging.